src/app/util/client-api.cpp

```diff
@@ -181,12 +181,16 @@ static uint16_t vFillBuffer(uint8_t * buffer, uint16_t bufferLen, uint8_t frameC
             {
                 valueLen = 4;
             }
+            else if (cmd == 'y')
```

Comment:
Can you update the comment in line 163 to 166 to reflect this addition?  Thanks.

Reply:
done